.github/workflows/release.yaml

```diff
@@ -8,22 +8,43 @@ on:
         description: 'Tag to be released'
         required: true
 
+env:
+  REGISTRY_NAME: registry.scality.com
+  PROJECT_NAME: ${{ github.event.repository.name }}
+
 jobs:
-  build-federation-image:
-    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
-    secrets: inherit
-    with:
-      push: true
-      registry: registry.scality.com
-      namespace: ${{ github.event.repository.name }}
-      name: ${{ github.event.repository.name }}
-      context: .
-      file: images/svc-base/Dockerfile
-      tag: ${{ github.event.inputs.tag }}-svc-base
```

Comment:
Confirm this with @scality/sup-setup, but I think you should keep this job, even if we won't be using 8.x on federation anytime soon you can build and push both docker images on each release.

But if you don't want to keep it then delete the image as well `images/svc-base/Dockerfile`